client/src/LoadClientDetails/styles.css

```diff
@@ -0,0 +1,31 @@
+.popup-box {
+    position: fixed;
+    background: #00000050;
+    width: 100%;
+    height: 100vh;
+  }
+  .box {
+    position: relative;
+    margin: 50px 70% 10px 15px;
+    height: auto;
+    max-height: 70vh;
+    background: #fff;
+    border-radius: 4px;
+    padding: 20px;
+    border: 5px solid #000;
+    overflow: auto;
+  }
+  .close-icon {
+    content: 'x';
```

Comment:
I noticed there is an x between the spans in LoadClientDetails/index.js. What happens with this content: 'x'?

Reply:
looking at it again it seems unnecessary